### PR TITLE
Say what to hide when you set cookies client side

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Fixes [#1849](https://github.com/alphagov/govuk-design-system/issues/1849).

Currently, [option 3 ('If you set non-essential cookies, but only on the client') of our cookie banner guidance](https://design-system.service.gov.uk/components/cookie-banner/#:~:text=When%20the%20page%20loads%2C%20include%20all%20the%20cookie%20banner%20messages%20that%20the%20user%20might%20see%2C%20but%20hide%20these%20with%20the%20hidden%20HTML%20attribute.) reads:

> When the page loads, include all the cookie banner messages that the user might see, but hide these with the hidden HTML attribute.

However, it wasn't clear to a user that both the component and the messages need to be hidden when the page loads. So this PR specifies what containers to hide.

### CODE REVIEW

**Review by lfdebrux — DISMISSED** [review]

File: src/components/cookie-banner/index.md.njk [hunk 1]
```diff
@@ -106,8 +106,8 @@ Here's the same example of a progressively enhanced cookie banner, with the conf
 ### Option 3: If you set non-essential cookies, but only on the client
 
 You can choose to make your banner only work with JavaScript if your service only needs to set non-essential cookies on the client.
- 
-When the page loads, include all the cookie banner messages that the user might see, but hide these with the `hidden` HTML attribute.
+
+When the page loads, hide both the component (`govuk-cookie-banner`) and all the cookie banner messages (`govuk-cookie-banner__message`) the user might see, using the `hidden` HTML attribute.
```
Comment (lfdebrux) [inline src/components/cookie-banner/index.md.njk:110]: I'm not sure about this... I find it a little confusing.

I think in part it might be that having the verb 'hide' implies that an action is taken, where in fact it is a lack of action that keeps the cookie banner hidden.

Comment (lfdebrux) [reply src/components/cookie-banner/index.md.njk:110]: Also, I'm not sure how helpful the class names are; while both these classes do need to be hidden, my understanding is that `govuk-cookie-banner__message` should always be inside a `govuk-cookie-banner`, so you shouldn't need the `hidden` attribute on both?

I could be wrong about that though.

Comment (EoinShaughnessy) [reply src/components/cookie-banner/index.md.njk:110]: Thanks for reviewing!

I think Hanna's concern was that users might not understand that, by 'component', we mean `govuk-cookie-banner`, and by 'message', we mean `govuk-cookie-banner__message`. But I'm not sure whether that's something a user reported or just a hunch about something that _could_ happen.

Would rewording like this avoid implying the user takes an action?

> When the page loads, the `hidden` html attribute hides the component, as well as all the cookie banner messages it contains, which the user might otherwise see.

Comment (EoinShaughnessy) [reply src/components/cookie-banner/index.md.njk:110]: @calvin-lau-sig7 Can I get a review? :)

Comment (lfdebrux) [reply src/components/cookie-banner/index.md.njk:110]: The first re-wording reads better to me!

Comment (EoinShaughnessy) [reply src/components/cookie-banner/index.md.njk:110]: Cheers, @lfdebrux!

**Review by calvin-lau-sig7 — APPROVED** [review]

**Comment by EoinShaughnessy** [comment]
@calvin-lau-sig7 Thanks for reviewing!
